web_src/js/features/repo-issue.js

```diff
@@ -87,21 +87,55 @@ export function initRepoIssueDue() {
 }
 
 export function initRepoIssueSidebarList() {
+  initIssueSearchDropdown('#new-dependency-drop-list');
+
+  function excludeLabel(item) {
+    const href = $(item).attr('href');
+    const id = $(item).data('label-id');
+
+    const regStr = `labels=((?:-?[0-9]+%2c)*)(${id})((?:%2c-?[0-9]+)*)&`;
+    const newStr = 'labels=$1-$2$3&';
+
+    window.location = href.replace(new RegExp(regStr), newStr);
+  }
+
+  $('.menu a.label-filter-item').each(function () {
+    $(this).on('click', function (e) {
+      if (e.altKey) {
+        e.preventDefault();
+        excludeLabel(this);
+      }
+    });
+  });
+
+  $('.menu .ui.dropdown.label-filter').on('keydown', (e) => {
+    if (e.altKey && e.keyCode === 13) {
+      const selectedItems = $('.menu .ui.dropdown.label-filter .menu .item.selected');
+      if (selectedItems.length > 0) {
+        excludeLabel($(selectedItems[0]));
+      }
+    }
+  });
+  $('.ui.dropdown.label-filter, .ui.dropdown.select-label').dropdown('setting', {'hideDividers': 'empty'}).dropdown('refreshItems');
+}
+
+function initIssueSearchDropdown(selector) {
```

Comment:
Can we not pass selectors but actual jQuery elements please? In this case I would name it `$list`.

Reply:
done in [1757d03](https://github.com/go-gitea/gitea/pull/25362/commits/1757d03280b4dbf6fdf6372285594d7ec9a91096)